octobot_trading/exchanges/exchange_manager.py

```diff
@@ -205,8 +205,8 @@ def get_exchange_credentials(self, logger, exchange_name):
 
     @staticmethod
     def handle_token_error(error, logger):
```

Comment:
I think we should move this method in ccxt_exchange or abstract_exchange

Reply:
I move it to abstract_exchange